### ISSUE TITLE
Packet loss causes websocket error

### ISSUE THREAD

**Issue by Temetz** [N4SJAMK/teamboard-feedback#96]
How to replicate:
1. Go to: http://noise.n4sjamk.org/boards/555afcade0ca9a2200cb586a/access/ace67716
2. Make sure that noise.n4sjamk.org is set up to drop inbound and outbound packets.
3. Move tickets around until you get logged out. Sometimes you may need to move only few tickets, sometimes a lot of tickets.

![image](https://cloud.githubusercontent.com/assets/8721037/7709116/7247d212-fe63-11e4-9fdd-7e7523f942b9.png)

```
Request Headers:
GET /api/boards HTTP/1.1
Host: noise.n4sjamk.org
Connection: keep-alive
Accept: application/json
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2398.0 Safari/537.36
Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6ImFhM2QxOWYyIiwidHlwZSI6Imd1ZXN0IiwiYWNjZXNzIjoiNTU1YWZjYWRlMGNhOWEyMjAwY2I1ODZhIiwidXNlcm5hbWUiOiJhc2RhZGFkYWRhZGFhZGFzZCIsImFjY2Vzc0NvZGUiOiJhY2U2NzcxNiIsImlhdCI6MTQzMjA1NDM5MX0._pJHAcWY63o_cqYP8-AhTnGrF60uN3-2lFa62gR0u5Q
Content-Type: application/json
Referer: http://noise.n4sjamk.org/boards/555afcade0ca9a2200cb586a
Accept-Encoding: gzip, deflate, sdch
Accept-Language: fi-FI,fi;q=0.8,en-US;q=0.6,en;q=0.4

---
Response Headers:
HTTP/1.1 401 Unauthorized
X-Powered-By: Express
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: x-access-token
WWW-Authenticate: Bearer realm="Users", error="invalid_token", error_description="User not found"
Content-Type: application/json; charset=utf-8
Content-Length: 79
ETag: W/"4f-3015306298"
Date: Tue, 19 May 2015 16:57:44 GMT

---
Response:
{
  "statusCode": 401,
  "error": "Unauthorized",
  "message": "Unauthorized"
}
```

```
Request Headers:
GET /socket.io/?access-token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6ImFhM2QxOWYyIiwidHlwZSI6Imd1ZXN0IiwiYWNjZXNzIjoiNTU1YWZjYWRlMGNhOWEyMjAwY2I1ODZhIiwidXNlcm5hbWUiOiJhc2RhZGFkYWRhZGFhZGFzZCIsImFjY2Vzc0NvZGUiOiJhY2U2NzcxNiIsImlhdCI6MTQzMjA1NDM5MX0._pJHAcWY63o_cqYP8-AhTnGrF60uN3-2lFa62gR0u5Q&EIO=3&transport=polling&t=1432054650517-5&sid=5ZyLnW1V_ztPWd98AAMP HTTP/1.1
Host: noise.n4sjamk.org
Connection: keep-alive
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2398.0 Safari/537.36
Accept: */*
Referer: http://noise.n4sjamk.org/boards/555afcade0ca9a2200cb586a
Accept-Encoding: gzip, deflate, sdch
Accept-Language: fi-FI,fi;q=0.8,en-US;q=0.6,en;q=0.4
Cookie: io=5ZyLnW1V_ztPWd98AAMP

---
Response Headers:
HTTP/1.1 400 Bad Request
Content-Type: application/json
Access-Control-Allow-Origin: *
Date: Tue, 19 May 2015 16:58:14 GMT
Transfer-Encoding: chunked

---
Response:
{"code":1,"message":"Session ID unknown"}
```

**Comment by tanelih** [comment]
Good find.

**Comment by Temetz** [comment]
I think this was fixed in N4SJAMK/teamboard-api@e9f7a78ee57328b940609643e28ea005c98fd9cf

**Comment by tanelih** [comment]
Ok, closing this for now then. If the problem reappears we'll reopen the issue.
